CHARTER.md

```diff
@@ -1,51 +1,123 @@
+  - Help community members on any of the official Arcalot channels
+
+### ART Participation
+
+If you wish to join the ART, please submit a pull request adding yourself to the
+[`ART_MEMBERS.md`](ART_MEMBERS.md) file. Please describe your use case in the pull
+request comments; you do not need to mention any company names or details. Joining is
+finalized upon two pull request approvals by active chairs and the merging of the pull
+request.
+
+Community meetings are not compulsory as we want to include people in all stages of life
+and across all timezones. We therefore refrain from any synchronous or undocumented
+decision making.
+
+As stated in the [Arcalot Code of
+Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md), chairs can
+remove any ART member if their behavior is clearly disruptive to the community.
+
+
+### Chair Participation
+
+Chairs are expected to remain actively involved in the community. A chair who does not
+participate in ART voting *without reason and notice* three times in a row forfeits the
+position and becomes a regular ART member automatically.
+
+
+## Proposals
+
+Proposals may only be made asychronously by submitting a pull request to this repository
+with a new file in the [`art-decisions/proposals`](art-decisions/proposals) directory.
+The provided [template file](art-decisions/proposals/template.md) may be used as a
+guideline for new proposals. The proposal file name should start with the date of the
+initial draft of the proposal in YYYY-MM-DD format, followed by the proposal title in
+all-lowercase with dashes substituting spaces (i.e., `2024-07-17-my-proposal-title.md`).
+Proposals may only be submitted by current ART members. If you would like to encourage
+discussion around a proposal before any voting starts, please submit a draft pull
+request instead. Once the discussion is complete and you are comfortable that you will
+not make any changes to your proposal, mark the pull request ready for review. Open pull
+requests can be viewed at the [ART repository pull
+requests](https://github.com/arcalot/arcalot-round-table/pulls) page.
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a 66% majority with a quorum of 66% of all ART members is needed for the proposal to pass.
-
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 2 months after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+Each member of the ART is eligible to vote if they have been a member for at least 30
+days. For any proposal that impacts the whole project or processes, **approval requires
+a minimum of four votes with 75% approval among those voting**.
```

Comment:
This text implies that there should also be a process or threshold for proposals which do not impact the whole project or its processes...but I don't remember seeing that.

Should we remove the qualifying phrase and just go with "Approval requires..."?

Reply:
I'm afraid we are going down a rabbit hole if we get too specific here, potentially leading to paragraphs more to define explicitly what "impacts the whole project" means and doesn't mean, and those paragraphs will have errors or complications, and we will be here reviewing charter changes in-perpetuity. I'm personally OK with a tight-knit org providing guidance on a case-by-case basis to the community on this.

Reply:
> I'm afraid we are going down a rabbit hole if we get too specific here

Understood.

> I'm personally OK with a tight-knit org providing guidance on a case-by-case basis to the community on this.

I don't have much of a problem with that.

The problem I have is that there is no process presented for determining whether a given proposal falls into the whole-project bucket _and_ there is no process presented for handling proposals which fall outside that bucket.

Rather than trying to provide those things, I suggest that we simply remove the phrase, "For any proposal that impacts the whole project or processes", which will remove the dichotomy.

If you really want to keep the explicit dichotomy (e.g., to support the case-by-case handling), then we need to add a statement here which says something like, "Proposals with lesser impact are handled by a process appointed by the chairs on a case-by-case basis".

Reply:
Without the dichotomy, as fuzzy as it may be, you can be left with the implication that *any* decision of *any* size or scope would need to go through this process, which we absolutely do not want.

For the non-proposal-process case, there is already legacy language in the charter:

Reply:
I think that the sentence before the one that you cite,
>This charter covers **important processes and decisions that impact the whole
organization, its direction, and its goals**.

potentially makes things clear, without having to repeat it in the section where we are struggling.